data/channels/inotterwords.yml

```diff
@@ -1,8 +1,15 @@
-description: One otter sets out to convince others of his positions on ethics and
-  politics, and perhaps be convinced of other positions himself!
+description:  One otter sets out to convince others of his positions on ethics and politics, 
+  and perhaps be convinced of other positions himself! 
+  My music www.wittynotes.com
```

Comment:
can this be put as a provider so that it comes out as a link.

Reply:
I had tried, but it didn't work, so that's why I had just opted for writing it in the description.